src/main/java/seedu/address/logic/parser/CommandList.java

```diff
@@ -28,7 +29,7 @@ public CommandList() {
         //add all commands to the list lexicographically
         commandList.add(AddCommand.COMMAND_WORD);
         commandList.add(ClearCommand.COMMAND_WORD);
-        commandList.add(ClearCommand.COMMAND_WORD);
```

Comment:
Is ClearCommand same as ClearHistoryCommand?

Reply:
Oh didn't see this, please merge after doubt cleared